Libplanet.Net/Consensus/ConsensusContext.cs

```diff
@@ -452,20 +452,28 @@ private Context CreateContext(long height)
         {
             // blockchain may not contain block of Height - 1?
             ValidatorSet validatorSet;
-            if (_blockChain.Tip.ProtocolVersion
-                < BlockMetadata.SlothProtocolVersion)
+            if (_blockChain.Tip.ProtocolVersion < BlockMetadata.SlothProtocolVersion)
             {
                 validatorSet = _blockChain
                     .GetWorldState(_blockChain[Height - 1].StateRootHash)
                     .GetValidatorSet();
             }
             else
             {
-                validatorSet = _blockChain
-                    .GetWorldState(
-                        _blockChain.GetNextStateRootHash(
-                            _blockChain[Height - 1].Hash, TimeSpan.Zero))
-                    .GetValidatorSet();
+                while (true)
+                {
+                    var nextStateRootHash = _blockChain.GetNextStateRootHash(Height - 1);
```

Comment:
Same as above.
```suggestion
                    var nextStateRootHash = _blockChain.GetNextStateRootHash(height - 1);
```

Reply:
Fixing this properly requires further rework and will be handled in a separate PR addressing #3833.

Reply:
👌